src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java

```diff
@@ -229,26 +230,22 @@ public void deleteApplication(String project, String application) throws IOExcep
 
     @Override
     public void push(String project, String application, String context, String component) throws IOException {
-        ExecHelper.executeWithTerminal(this.project, WINDOW_TITLE, new File(context), true, envVars, command, "push");
+        ExecHelper.executeWithTerminal(this.project, WINDOW_TITLE, new File(context), true, envVars, null, null, null, command, "push");
     }
 
     @Override
     public void pushWithDebug(String project, String application, String context, String component) throws IOException {
-        ExecHelper.execute(command, true, new File(context), envVars, "push", "--debug");
+        ExecHelper.execute(command, true, new File(context), envVars, null, null, null, "push", "--debug");
     }
 
     @Override
     public void describeComponent(String project, String application, String context, String component) throws IOException {
-        if (context != null) {
-            ExecHelper.executeWithTerminal(this.project, WINDOW_TITLE, new File(context), false, envVars, command, "describe");
-        } else {
-            ExecHelper.executeWithTerminal(this.project, WINDOW_TITLE, false, envVars, command, "describe", "--project", project, "--app", application, component);
-        }
+        ExecHelper.executeWithTerminal(this.project, WINDOW_TITLE, createWorkingDirectory(context), false, envVars, null, null, null, command, "describe");
```

Comment:
@sbouchet, @jeffmaury, @lstocchi: please verify this fix for breaking changes in intellij-common